mappings/net/minecraft/server/player/PlayerManager.mapping

```diff
@@ -0,0 +1,98 @@
+CLASS net/minecraft/class_166 net/minecraft/server/player/PlayerManager
+	FIELD field_577 logger Ljava/util/logging/Logger;
+	FIELD field_578 players Ljava/util/List;
+	FIELD field_579 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_580 chunkTrackers [Lnet/minecraft/class_167;
+	FIELD field_581 maxPlayerCount I
+	FIELD field_582 bannedPlayers Ljava/util/Set;
+	FIELD field_583 ipBans Ljava/util/Set;
+	FIELD field_584 operators Ljava/util/Set;
+	FIELD field_585 whitelist Ljava/util/Set;
+	FIELD field_586 bannedPlayersFile Ljava/io/File;
+	FIELD field_587 bannedIpsFile Ljava/io/File;
```

Comment:
ipBans -> bannedIps or bannedIpsFile -> ipBansFile

Reply:
it's fine but consistency

Reply:
I change my mind this is not fine this will actually destroy the world with godzilla superman-lazer anime power-of-friendship harambe monke power if not fixed